.circleci/config.yml

```diff
@@ -40,12 +40,12 @@ commands:
               --volume=$HOME/.cache/go-build/:/root/.cache/go-build \
               --volume=$HOME/go/pkg/mod/:/go/pkg/mod \
               --volume=$HOME/project/.ssh:/root/.ssh \
-              weaveworks/eksctl-build:$(cat .docker/image_tag) << parameters.cmd >>
+              sayboras/eksctl-build:$(cat .docker/image_tag) << parameters.cmd >>
```

Comment:
These changes for testing purpose only as I don't have permission to push images into weaveworks docker hub